api/migrations/024_fix_template_schema.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Nanocloud turns any traditional software into a cloud solution, without
+ * changing or redeveloping existing source code.
+ *
+ * Copyright (C) 2016 Nanocloud Software
+ *
+ * This file is part of Nanocloud.
+ *
+ * Nanocloud is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Nanocloud is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General
+ * Public License
+ * along with this program.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+
+function up(knex, Promise) {
+  return knex.schema.table('template', (table) => {
+    // There is no method in knex for datatype changes.
+    // https://github.com/tgriesser/knex/issues/46
+    table.renameColumn('content', 'old_content');
+  })
+    .then(() => {
+      return knex.schema.table('template', (table) => {
+        table.text('content');
+      });
+    })
+    .then(() => {
+      return knex('template');
+    })
+    .then((templates) => {
+      return Promise.map(templates, function(template) {
+        return knex('template').where({
+          id: template.id
+        }).update({
+          content: template.old_content
+        });
+      });
+    })
+    .then(() => {
+      return knex.schema.table('template', (table) => {
+        table.dropColumn('old_content');
+      });
+    });
+}
+
+function down(knex) {
+  return knex.schema.dropTable('template');
```

Comment:
The `down` function should describe the way to downgrade the change. Here it's just destroying the date.